scripts/Elemental_Analysis.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import, print_function
+
+from PyQt4 import QtGui
+
+import sys
+
+from Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table_presenter import PeriodicTablePresenter
+from Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table_view import PeriodicTableView
+from Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table_model import PeriodicTableModel
+from Muon.GUI.Common import message_box
+
+
+class ElementalAnalysisGui(QtGui.QMainWindow):
+    def __init__(self, parent=None):
+        super(ElementalAnalysisGui, self).__init__(parent)
+
+        self.ptable = PeriodicTablePresenter(
+            PeriodicTableView(), PeriodicTableModel())
+        self.ptable.register_table_changed(self.table_changed)
+        self.ptable.register_table_lclicked(self.table_left_clicked)
+        self.ptable.register_table_rclicked(self.table_right_clicked)
+        self.setCentralWidget(self.ptable.view)
+        self.setWindowTitle("Elemental Analysis")
+
+    def table_left_clicked(self, item):
+        print("Element Left Clicked: {}".format(
```

Comment:
Are these print statements the right thing to do now that this is going into the nightly build?

Reply:
yes, there are only two people who will be using the interface and they know that it just prints to the logger for now.

Reply:
👍